### PR TITLE
[IntelliJ Plugin] Enable spell checker

### LINKED ISSUE #10499: [IDEA] Enable spell checking
**Description:**
Spell checking is disabled at the moment. Need to re-enable spell checking.

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
This PR enables spell checker capability in ballerina IDEA plugin.

Fixes #10499

### CODE REVIEW

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaIdentifierTokenizer.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ */
+public class BallerinaIdentifierTokenizer extends Tokenizer<PsiNameIdentifierOwner> {
+
+    @Override
+    public void tokenize(@NotNull PsiNameIdentifierOwner element, TokenConsumer consumer) {
+        // Do nothing here. This is used to prevent adding multiple fix suggestions (Rename To, etc) to the code.
+        // Otherwise one suggestion will be added when the identifier is processed, another will be added when the
+        // parent node of the identifier is processed.
+    }
+}
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaIdentifierTokenizer.java]: Shall we add a newline here.

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaSpellcheckingStrategy.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        if (element instanceof BallerinaIdentifier) {
+            if (((BallerinaIdentifier) element).getElementType() == BallerinaTypes.IDENTIFIER) {
+                return identifierTokenizer;
+            }
+        }
+        return super.getTokenizer(element);
+    }
+
+    // Todo - Add support for markdown documentation and line comments.
+}
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaSpellcheckingStrategy.java]: Missing newline here as well.

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaIdentifierTokenizer.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaIdentifierTokenizer.java]: Added

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaSpellcheckingStrategy.java (hunk 2 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/spellchecker/BallerinaSpellcheckingStrategy.java]: Added

**Pull request merged**